openapi.yaml

```diff
@@ -5921,6 +5921,665 @@ paths:
+                https://api.linode.com/v4beta/lke/clusters/12345
+      - lang: CLI
+        source: >
+          linode-cli lke cluster-delete 12345
+  /lke/clusters/{clusterId}/pools:
+    parameters:
+      - $ref: '#/components/parameters/clusterId'
+    x-linode-cli-command: lke
+    get:
+      operationId: getLKEClusterPools
+      x-linode-cli-action: pools-list
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - lke:read_only
+      tags:
+      - Linode Kubernetes Engine (LKE)
+      servers:
+      - url: https://api.linode.com/v4beta
+      summary: List Node Pools
+      description: >
+        Returns all active Node Pools on a Kubernetes cluster.
+
+
+        **Beta**: This endpoint is in beta. Please make sure to prepend all requests with
+        `/v4beta` instead of `/v4`, and be aware that this endpoint may receive breaking
+        updates in the future. This notice will be removed when this endpoint is out of
+        beta.
+      responses:
+        '200':
+          description: Returns an array of all Pools in this Kubernetes cluster.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/LKENodePool'
+                  page:
+                    $ref: '#/components/schemas/PaginationEnvelope/properties/page'
+                  pages:
+                    $ref: '#/components/schemas/PaginationEnvelope/properties/pages'
+                  results:
+                    $ref: '#/components/schemas/PaginationEnvelope/properties/results'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+                https://api.linode.com/v4beta/lke/clusters/12345/pools
+      - lang: CLI
+        source: >
+          linode-cli lke pools-list 12345
+    post:
+      operationId: postLKEClusterPools
+      x-linode-cli-action: pool-create
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - lke:read_write
+      tags:
+      - Linode Kubernetes Engine (LKE)
+      servers:
+      - url: https://api.linode.com/v4beta
+      summary: Create Node Pool
+      description: >
+        Creates a new Node Pool for the designated Kubernetes cluster.
+
+
+        **Beta**: This endpoint is in beta. Please make sure to prepend all requests with
+        `/v4beta` instead of `/v4`, and be aware that this endpoint may receive breaking
+        updates in the future. This notice will be removed when this endpoint is out of
+        beta.
+      requestBody:
+        description: Configuration for the Node Pool
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/LKENodePoolRequestBody'
```

Comment:
I think both type and count need to be required here see lines 312-318 of apinext/apinext/blueprints/beta/lke_clusters.py